chromium_src/components/metrics/machine_id_provider_win.cc

```diff
@@ -0,0 +1,13 @@
+#include "base/command_line.h"
+#include "brave/common/brave_switches.h"
+
+// switches::kDisableMachineId
+const char kDisableMachineId[] = "disable-machine-id";

```

Comment:
ok, this is seriously the last thing, I just didn't notice until now that kDisableMachineId and kDisableEncryptionWin are defined twice (once here and once in brave_switches.cc). I'm surprised that builds without errors, but can we remove it here?

Reply:
This is part of the discussion [here](https://github.com/brave/brave-core/pull/795#discussion_r247298713), I cannot manage to use switches from brave_switches.h :(

Reply:
```
ninja -t msvc -e environment.x64 -- ../../third_party/llvm-build/Release+Asserts/bin/lld-link.exe /nologo /IMPLIB:./os_crypt.dll.lib /DLL /OUT:./os_crypt.dll /PDB:./os_crypt.dll.pdb @./os_crypt.dll.rsp
lld-link: error: undefined symbol: "char const *const switches::kDisableEncryptionWin" (?kDisableEncryptionWin@switches@@3QBDB)
>>> referenced by C:\brave\src\brave\chromium_src\components\os_crypt\os_crypt_win.cc:9
>>>               obj/components/os_crypt/os_crypt/os_crypt_win.obj:("bool __cdecl `anonymous namespace'::IsEncryptionDisabled(class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>, class basic_string<char, struct std::char_traits<char>, class std::allocator<char>>::0x1A85D7CC *)" (?IsEncryptionDisabled@?A0x1A85D7CC@@YA_NV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAV12@@Z))
[3 processes, 8/246 @ 3.6/s : 2.193s ] CXX obj/brave/components/brave_sync/brave_sync/brave_sync_service.obj
```

Reply:
that's so weird, I would think that you would get exactly the opposite issue of having a duplicate. I'll build and take a quick look.

Reply:
I think that's why @darkdh done this [here](https://github.com/brave/brave-core/commit/0c1a3b676f0e00d467c7ec1662fd388de36832bf#diff-68b1ac1b9b49edcc9e39fe595e8cef2eR10).

Reply:
ok, I'm just going to approve then since @darkdh's similar code is not causing any problems for any of the builds

Reply:
Ok then!